telemetry/lib/LqCalculator/LqCalculator.hpp

```diff
@@ -77,8 +74,8 @@ class LqCalculator {
     // after a failsafe, instead of down from 100
     LQ = 0;
     index = 0;
-    LQmask = (1 << 0);
-    for (uint8_t i = 0; i < (sizeof(LQArray) / sizeof(LQArray[0])); i++) {
+    LQmask = (1U << 0U);
+    for (uint32_t i = 0; i < (sizeof(LQArray) / sizeof(LQArray[0])); i++) {
```

Comment:
Why to uint32_t?

Reply:
Because `sizeof(LQArray) / sizeof(LQArray[0])` returns an uint32_t element. In reality this doesn't matter as from what I see LQArray's max size is 2, but I didn't want to change the semantics of the line in this patch.